.eslintrc

```diff
@@ -43,7 +43,7 @@
       "never"
     ],
     "one-var": [
-      2,
+      0,
```

Comment:
What this would mean is declaring all functions at the very top of the file.

Right now the functions are split very nicely according to functionality - example: the drag event listener and functions are [sitting together](https://github.com/esfiddle/esfiddle/pull/81/files#diff-c862590c41b82dbf97e9a329826ed958R388), the options to customize the interface are [sitting together](https://github.com/esfiddle/esfiddle/pull/81/files#diff-c862590c41b82dbf97e9a329826ed958R102). That's very useful to keep.